portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,101 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns data of comments*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    public class Comment { 
+        String name; 
+        String email;
+        String comment;
+        long timestamp; 
+        long id;
```

Comment:
Do you expect to need to change these fields after the object is constructed? If not, it's better to make it so you know for sure that they won't change by marking them as `final`. Additionally, having no access modifier is itself an access modifier which provides access that is somewhere between `private` and `protected`. If that's what you intended, that's fine, but you should be aware of what it means and make sure you are choosing an appropriate access modifier.

Note that the Gson documentation says "It is perfectly fine (and recommended) to use private fields.", so you should be able to just make these private.

Reply:
done